flexget/components/sites/sites/ncore.py

```diff
@@ -111,15 +112,15 @@ def search(self, task, entry, config):
                     href = a.get('href')
                     id = href[href.find("id=") + 3 :]
 
-                    e['title'] = a.text
+                    e['title'] = a.get('title')
                     e['url'] = URL + f'/torrents.php?action=download&id={id}&' + PASSKEY
 
                     parent = a.parent.parent.parent.parent
 
-                    e['torrent_seeds'] = parent.find('div', class_='box_s2').string
-                    e['torrent_leeches'] = parent.find('div', class_='box_l2').string
-                    e['content_size'] = (
-                        float(parent.find('div', class_='box_meret2').string) * 1024**2
+                    e['torrent_seeds'] = int(parent.find('div', class_='box_s2').string)
+                    e['torrent_leeches'] = int(parent.find('div', class_='box_l2').string)
+                    e['content_size'] = parse_filesize(
+                        parent.find('div', class_='box_meret2').string
```

Comment:
The `parse_filesize` function expects there to be a unit in the string. The previous code looks like it was just expecting a bare number. Did you test this works?

Reply:
Yep, I tested it, and there is unit in the string. It was not tested when it was created.